pkgs/leak_tracker_flutter_testing/lib/src/leak_tracking_for_tests.dart

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:leak_tracker/leak_tracker.dart';
+import 'package:meta/meta.dart';
+
+void _emptyLeakHandler(Leaks leaks) {}
+
+/// Set of helper methods to adjust default leak tracker settings for tests.
+///
+/// Some metods modify [LeakTrackingForTests.settings], others return adjusted
+/// copy of [LeakTrackingForTests.settings].
+///
+/// Use methods, which modify [LeakTrackingForTests.settings], to
+/// change default settings for set of tests,
+/// for example for package or folder (in flutter_test_config.dart) or
+/// for library (in `setUpAll`).
+///
+/// Use methods that return adjusted [LeakTrackingForTests.settings]
+/// to customize default for an individual test:
+///
+/// ```dart
+/// testWidgets(
+///     'initialTimerDuration falls within limit',
+///     leakTracking: LeakTrackingForTests.withIgnoredLeaks(allNotGCed: true),
+///     (WidgetTester tester) async {
+///       ...
+/// ```
+///
+/// Update for [LeakTrackingForTests.settings] when a test is already running
+/// will throw exception.
```

Comment:
This doesn't seem to match the implementation since settings is just a field?

Reply:
For now nothing matches implementation, because leak tracker does not use the settings yet.
It is work in progress.

Reply:
Updated comment.